README.md

```diff
@@ -2,10 +2,10 @@
 
 **Beta-quality. See [the last section of this README](#trustworthiness).**
 
-Bootsnap is a library that plugs into a number of Ruby and (optionally) `ActiveSupport` and `YAML`
-methods to optimize and cache expensive computations. See [the How Does This Work section](#how-does-this-work) for more information.
+Bootsnap is a library that can plug into Ruby, `ActiveSupport` and `YAML`
```

Comment:
We want to emphasize that ActiveSupport is optional. Rails is not a dependency for Bootsnap